dashboard/src/app/workspaces/workspace-status-indicator/workspace-status-indicator-circle.directive.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * Defines a directive for creating simple circle indicator of workspace's status.
+ * @author Oleksii Kurinnyi
+ */
+export class WorkspaceStatusIndicatorCircle {
```

Comment:
WorkspaceStatusIndicatorCircle -> WorkspaceStatusIndicator cause can be not only circle (in progress example)